build.gradle

```diff
@@ -9,18 +9,16 @@ buildscript {
     }
 }
 
-allprojects {
```

Comment:
The project does not compile for me without the `google` repo

Reply:
Re-added